packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -201,19 +221,48 @@ const { requiredMarkerClass, requiredTypeClass } = useRequired(props);
     }
   }
 
-  &:has(.onyx-select__input:enabled:focus) {
-    .onyx-select {
-      &__wrapper {
-        --border-color: var(--onyx-color-base-primary-500);
-        outline: var(--onyx-spacing-4xs) solid var(--onyx-color-base-primary-200);
+  &__loading {
+    color: var(--onyx-color-text-icons-primary-intense);
+  }
+
+  &__footer {
+    width: 100%;
+    color: var(--onyx-color-text-icons-neutral-soft);
+  }
+
+  &--editable {
```

Comment:
Would prefer to use the `:read-write` CSS pseudo selector

Reply:
The input is always readonly as it is only used for the presentation of the selected value